### PR TITLE
Add status callbacks

### PR DESCRIPTION
**Pull request by VadymMoiseyenkoAgiliway**
## Overview

For more advanced usage of a component, we will be very grateful if we can add "onError" and "onSuccess" callbacks as component props.

It will bring the possibility to change "cardSize" after fetching result (E.g. switch "showGraphic" to false if we don't have image URL) Also it can help add alert information for the user about not valid URL.

### CODE REVIEW

File: src/ReactTinyLink.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ const fetchUrl = (
   defaultMedia: string,
   setData: (data: IReactTinyLinkData) => void,
   setLoading: (loading: boolean) => void,
+  onError: Function,
```
Comment (winhtaikaung) [inline src/ReactTinyLink.tsx]: Would it be great  if we can have `error` as a param like this ? 
`onError:(error: Error) => void,`

File: src/ReactTinyLink.tsx [hunk 2]
```diff
@@ -17,6 +17,8 @@ const fetchUrl = (
...
+  onSuccess: Function,
```
Comment (winhtaikaung) [inline src/ReactTinyLink.tsx]: Would it be great  if we can have `response` as a param like this ? 
`onSuccess:(response: Response) => void,`

**Comment by winhtaikaung** [comment]
Hi @VadymMoiseyenkoAgiliway, awesome work  and thank you for your contribution,
 There are some request from me to pass parameter for those methods.
:+1:

**Comment by VadymMoiseyenkoAgiliway** [comment]
Hi @winhtaikaung! Thank you for your feedback 😀

I've updated PR, but changed a bit your suggestion.
Instead 
`onSuccess:(response: Response)` 
I used
`onSuccess:(response: IReactTinyLinkData)` 
because in `ScraperWraper` we receive `IReactTinyLinkData` as data type, and using `Response` caused error

**Comment by winhtaikaung** [comment]
Adds @VadymMoiseyenkoAgiliway  as a contributor for code.

**Comment by winhtaikaung** [comment]
@all-contributors please add  @VadymMoiseyenkoAgiliway  for code bug

**Pull request merged**